src/routes/dashboard/(admin)/admin/transactions/+page.svelte

```diff
@@ -338,7 +338,7 @@
                         href="#"
                         on:click={show_generate_modal(transaction)}
                         class="font-medium pr-2 text-green-600 dark:text-green-400 hover:underline"
-                        >Generate</a
+                        >Download</a
```

Comment:
Based on the defect report the issue is for the `Report Generation` not with the `Document Generation`. I suggest to keep the button as `Generate`, the one that should be changed it this one: 
https://github.com/Toshiven/online-appointment-and-certificate-generator/blob/9d4bf84d3bad68c53c706d4e660229c2845c40c2/src/routes/dashboard/(admin)/admin/transactions/%2Bpage.svelte#L291-L296